avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -135,7 +136,8 @@ Describe 'File/folder tests' -Tag 'Modules' {
       if ($isOrphaned) {
         $pathExisting = Test-Path $orphanedFilePath
         $pathExisting | Should -Be $true -Because 'The module is orphaned.'
-      } else {
+      }
```

Comment:
Where thse changes intended? It appears a local format setting was applied to the file

Reply:
I did not change or add anything to this file. I am not sure about this piece of code.

Reply:
It seems that this is similar to what is there in the repo:
![image](https://github.com/Azure/bicep-registry-modules/assets/47571120/8e79e288-d1e3-4346-972a-a8207a5e6a8f)

Reply:
Hey @jeetgarg, it absolutely is. It's just the autoformatter that left an impression in your case ;) In general, it's not much of an issue - but if we were to merge it, GitHub would detect it as a change and re-run the workflows of all modules as it's a platform script. Unfortunately GitHub triggers are just not that smart 😄

Reply:
On our end we plan to implement custom vscode settings eventually to make sure this can't happen, but until now we unfortunately didn't have capacity to look into this.

Reply:
Feel free to ping me internally if you need help

Reply:
Check now. If its still not right, i will ping you

Reply:
One line got corrected, the other 95 not 😄 But progress. 
All the other fixes worked though.